linetools/spectra/lsf.py

```diff
@@ -230,17 +231,36 @@ def interpolate_to_wv0(self,wv0):
         #transform to wavelength in float() form assuming Angstroms
         col_names = self._data.keys()
         col_waves = [float(name.split('A')[0]) for name in col_names[1:]]
-                    
+
         lsf_vals = []
         for row in self._data:
             aux_val = []
             for i in range(1,len(row)):
                 aux_val += [row[i]]
-            f = interp1d(col_waves,aux_val,bounds_error=True,kind='linear') #we do not want to extrapolate
-            lsf_vals += [f(wv0.value)]
+            #we don't want to extrapolate wildly, but allow small extrapolations for wv0
+            if (wv0.value >= col_waves[0]) & (wv0.value <= col_waves[-1]):
+                f = interp1d(col_waves,aux_val,bounds_error=True,kind='linear') #no need to extrapolate
+                lsf_vals += [f(wv0.value)]
+
+            elif (wv0.value < col_waves[0]) & \
+                    ((col_waves[0]-wv0.value) < np.abs(col_waves[1]-col_waves[0])):
```

Comment:
Can you limit the extrapolation to `np.abs(col_waves[1]-col_waves[0])) / 2` , i.e. half way only.